apis/v1alpha1/validation_test.go

```diff
@@ -0,0 +1,145 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import (
+	"testing"
+
+	"github.com/abcxyz/pkg/testutil"
+)
+
+func TestValidateIAMRequest(t *testing.T) {
+	t.Parallel()
+
+	cases := []struct {
+		name    string
+		request *IAMRequest
+		wantErr string
+	}{
+		{
+			name: "success",
+			request: &IAMRequest{
+				ResourcePolicies: []*ResourcePolicy{
+					{
+						Resource: "organizations/foo",
+						Bindings: []*Binding{
+							{
+								Members: []string{
+									"user:test-org-userA@example.com",
+									"user:test-org-userB@example.com",
+								},
+								Role: "roles/accessapproval.approver",
+							},
+						},
+					},
+					{
+						Resource: "folders/bar",
+						Bindings: []*Binding{
+							{
+								Members: []string{
+									"user:test-folder-user@example.com",
+								},
+								Role: "roles/cloudkms.cryptoOperator",
+							},
+						},
+					},
+					{
+						Resource: "projects/baz",
+						Bindings: []*Binding{
+							{
+								Members: []string{
+									"user:test-project-user@example.com",
+								},
+								Role: "roles/bigquery.dataViewer",
+							},
+						},
+					},
+				},
+			},
+		},
+		{
+			name: "invalid email",
+			request: &IAMRequest{
+				ResourcePolicies: []*ResourcePolicy{
+					{
+						Resource: "organizations/foo",
+						Bindings: []*Binding{
+							{
+								Members: []string{
+									"user:example.com",
+								},
+								Role: "roles/accessapproval.approver",
+							},
+						},
+					},
+				},
+			},
+			wantErr: "email example.com is not valid",
+		},
+		{
+			name: "invalid member",
```

Comment:
Please review and follow https://github.com/abcxyz/readability/wiki/Go-Best-Practices#naming

The test name should not have spaces (use underscores instead).